test/hotspot/jtreg/compiler/vectorization/TestReplicateAtConv.java

```diff
@@ -26,6 +26,7 @@
  * @bug 8341834
  * @summary C2 compilation fails with "bad AD file" due to Replicate
  * @run main/othervm -XX:CompileCommand=compileonly,TestReplicateAtConv::test -Xcomp TestReplicateAtConv
+ * @run main/othervm -XX:CompileCommand=compileonly,TestReplicateAtConv::test -Xcomp -XX:MaxVectorSize=8 TestReplicateAtConv
```

Comment:
You should add the new bug id to the test.

Reply:
Maybe you should also update the summary, and say that the issue is about replicate and cast.

Reply:
Done in new commit.